velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -891,6 +891,14 @@ TEST_F(DateTimeFunctionsTest, fromUnixtime) {
   // 8 hours ahead UTC.
   setQueryTimeZone("Asia/Shanghai");
 
+#ifdef NDEBUG
```

Comment:
Also, it's sort of weird that we only let it overflow in release code and add a test to ensure it. The lack of test in release wasn't intentional, but due to performance to prevent the checking overhead. How could we make this more general?